webapp/src/compiler.ts

```diff
@@ -317,6 +317,8 @@ export function pySnippetToBlocksAsync(code: string, blockInfo?: ts.pxtc.BlocksI
 // Py[] -> blocks
 export function pySnippetArrayToBlocksAsync(code: string[], blockInfo?: ts.pxtc.BlocksInfo): Promise<pxtc.CompileResult> {
     const namespaceRegex = /^\s*namespace\s+[^\s]+\s*{([\S\s]*)}\s*$/im;
+    const exportLetRegex = /^\s*export\s(let\s*[\S\s]*)\s*$/im;
```

Comment:
Hm, it seems like this would break tutorials with custom spritekinds e.g. https://github.com/microsoft/pxt-arcade/blob/master/docs/tutorials/paddle.md? I might be missing something though

Reply:
ooh I'll try it. I only tested it with minecraft tutorials

Reply:
hmmm so it seems to work fine for the paddles ( I made a new python version of it to test ), but why do you think it would break tutorials? maybe I'm not testing rigorously enough

Reply:
Okay I pulled for a sec and checked and we don't hit it because it uses `const` instead of `let` -- the issue I was thinking of was that if we had something like:

```typescript
namespace SpriteKind {
    export const LeftPaddles = SpriteKind.create()
}
let left_paddle = sprites.create(img`
    a 8 1
    8 8 1
`, SpriteKind.LeftPaddles)
```

it would swap to

```typescript
namespace SpriteKind {
    const LeftPaddles = SpriteKind.create()
}
let left_paddle = sprites.create(img`
    a 8 1
    8 8 1
`, SpriteKind.LeftPaddles)
```

(but again those are `const`s not `let`s so it doesn't get hit here)

Reply:
ahhh but we don't add blocks in the toolbox for creating the custom spritekinds, right?

Reply:
Not blocks in toolbox, but the references to `SpriteKind.LeftPaddles` would be invalid when compiling

Reply:
hmm so I think this is only used to filter the toolbox? but maybe it's not clear and dangerous to put here?

Reply:
The issue would be if it fails to compile and get the blocks info for the toolbox. But since we always use const for our generated values inside namespaces, we won't hit that here -- maybe worth a note in a comment that we don't want to remove the export for consts but I don't think it's something to block this fix on :)

Reply:
sounds good I'll add that!